### DISCUSSION TITLE
How to configure the Controller for multi-tenancy

### DISCUSSION THREAD

**Discussion by mike-chen-samsung** [actions/actions-runner-controller#3176] [General]
The official docs do not address this question https://github.com/actions/actions-runner-controller/blob/master/docs/using-arc-across-organizations.md

When installing the ARC Helm Chart for multitenancy, how should the Controller be configured to auth to multiple Github Orgs given that the Chart only accepts a single set of credentials?

From what I understand, the Org creds are necessary for the controller to know which Runners to scale up, but if it only has access to one org's credentials, it won't know to scale up Runners for the other Orgs that it doesn't have credentials for.

There is some discussion in this thread but the question is ultimately not answered here either: https://github.com/actions/actions-runner-controller/discussions/1832

**Comment by MPV** [comment]
If you use enterprises, this could work:
- https://github.com/actions/actions-runner-controller/discussions/3071

**Reply by mike-chen-samsung** [reply]
Unfortunately I am limited to creating organization runners in my GHES instance. Is multi-tenancy only supported for Enterprise Runners?

**Reply by kaykhan** [reply]
@MPV @mike-chen-samsung Im also interested in a solution that does not require GitHub Enterprise Server

**Reply by MPV** [reply]
@kaykhan have you tried and disproven my theory that the linked suggestion wouldn't work for Enterprise Cloud too (and just Enterprise Server)?

**Reply by kaykhan** [reply]
@MPV No i have not, i meant to say we are looking for a solution that allows  multi-tenancy, something that would just work for regular Github Account (free) accounts
